src/UI/Settings/Notifications/Edit/NotificationEditViewTemplate.hbs

```diff
@@ -33,7 +33,7 @@
                         </label>
 
                         <span class="help-inline-checkbox">
-                            <i class="icon-nd-form-info" title="Do you want to get notifications when episodes are grabbed?"/>
+                            <i class="icon-nd-form-info" title="Be notified when episodes are available for download and has been sent to a download client"/>
```

Comment:
I think the available part is implied. we can't send something that isn't available to the download clients. so I suggest,

"Be notified when episodes are sent to a download client"